src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/PackageVulnerabilityService.cs

```diff
@@ -17,26 +18,26 @@ namespace NuGet.PackageManagement.VisualStudio
     public class PackageVulnerabilityService : IPackageVulnerabilityService
     {
         private readonly IEnumerable<SourceRepository> _sourceRepositories;
-        private GetVulnerabilityInfoResult _vulnerabilities;
+        private readonly AsyncLazy<GetVulnerabilityInfoResult> _vulnerabilitiesLazy;
         private INuGetUILogger _logger;
 
-        public PackageVulnerabilityService(IEnumerable<SourceRepository> sourceRepositories, INuGetUILogger logger)
+        public PackageVulnerabilityService(IEnumerable<SourceRepository> sourceRepositories, INuGetUILogger logger, CancellationToken cancellationToken)
```

Comment:
The constructor had a `CancellationToken` added to the parameter list, but it's not used anywhere.